.github/workflows/main.yml

```diff
@@ -65,6 +65,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.ref }}
+          submodules: recursive
```

Comment:
Does this job have `workflow_dispatch`?

Reply:
dang I forgot to remove this when debugging the github actions workflows with tar unpacking. Reverting this.